src/components/GeneralInfo/DataCollectorsCard/DataCollectorsCard.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable camelcase */
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import LoadingCard from '../LoadingCard';
+import DateFormat from '@redhat-cloud-services/frontend-components/DateFormat';
+import { TableComposable, Thead, Tr, Th, Tbody, Td, TableVariant, ExpandableRowContent } from '@patternfly/react-table';
+import { Flex, FlexItem } from '@patternfly/react-core';
+import { getDefaultCollectors } from '../selectors/selectors';
+
+const DataCollectorsCard = ({
+    detailLoaded,
+    collectors,
+    entity,
+    dataMapper
+}) => {
+    const [expandedNames, setExpandedNames] = useState([]);
+    const setExpanded = useCallback((collector, isExpanding = true) => setExpandedNames(prevExpanded => {
+        const otherExpandedNames = prevExpanded.filter(r => r !== collector.name);
+        return isExpanding ? [...otherExpandedNames, collector.name] : otherExpandedNames;
+    }), []);
+    const isExpanded = (collector) => expandedNames.includes(collector.name);
+    const data = collectors ?? getDefaultCollectors(entity);
+    return (<LoadingCard
+        title="Data collectors"
+        isLoading={ !detailLoaded }
+    >
+        <TableComposable
+            aria-label="Data collectors"
+            variant={TableVariant.compact}
+            borders={false}
+        >
+            <Thead>
+                <Tr className="ins-c__no-border">
+                    <Th />
+                    <Th>{'Name'}</Th>
+                    <Th>{'Status'}</Th>
+                    <Th>{'Last upload'}</Th>
```

Comment:
No need to wrap strings in object literals
```suggestion
                    <Th>Name</Th>
                    <Th>Status</Th>
                    <Th>Last upload</Th>
```